gcp_variant_transforms/testing/integration/run_tests_common.py

```diff
@@ -146,25 +162,31 @@ def get_configs(test_file_path, required_keys):
   for root, _, files in os.walk(test_file_path):
     for filename in files:
       if filename.endswith('.json'):
-        test_configs.append(_load_test_config(os.path.join(root, filename),
-                                              required_keys))
+        test_configs.append(_load_test_configs(os.path.join(root, filename),
+                                               required_keys))
   if not test_configs:
     raise TestCaseFailure('Found no .json files in directory {}'.format(
         test_file_path))
   return test_configs
 
 
-def _load_test_config(filename, required_keys):
-  # type: (str, List[str]) -> str
+def _load_test_configs(filename, required_keys):
+  # type: (str, List[str]) -> List[Dict]
   """Loads an integration test JSON object from a file."""
   with open(filename, 'r') as f:
-    test = json.loads(f.read())
-    _validate_test(test, filename, required_keys)
-    return test
+    tests = json.loads(f.read())
+    # When there are multiple test cases in one JSON file, it is a list. Wrap
+    # the dictionary in a list if there is only one test case in one JSON file.
+    if not isinstance(tests, list):
```

Comment:
Actually I prefer to modify all other JSON files be a list rather than wrapping it here. It would be more readable. WDYT?

Reply:
SGTM!